KSP-Backend/backend/api/views.py

```diff
@@ -0,0 +1,49 @@
+from django.shortcuts import render
+from rest_framework import viewsets
+from vote.views import VoteMixin
+from .serializers import FeedbackSerializer, NotesSerializer, PreviousYearQuestionsSerializer
+from .models import Feedback, Notes, PreviousYearQuestions
+
+# Create your views here.
+
+class FeedbackView(viewsets.ModelViewSet):
+    serializer_class = FeedbackSerializer
+    queryset = Feedback.objects.all()
+
+class NotesView(viewsets.ModelViewSet, VoteMixin):
+    serializer_class = NotesSerializer
+    queryset = Notes.objects.all()
+    #review = Notes.objects.get(pk=1)
+    #upvote
+    #review.votes.up(id)
+
+class PreviousYearQuestionsView(viewsets.ModelViewSet):
+    serializer_class = PreviousYearQuestionsSerializer
+    queryset = PreviousYearQuestions.objects.all()
+
+
+
+def vote(request, note_id):
+            notes = Notes.objects.get(pk=note_id)

```

Comment:
Nit/Pick: Fix the formatting here. In every other class/ function you are using a different number of spaces(or tabs) for indentation.